Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Melee/plasma_sword.yml

```
@@ -0,0 +1,213 @@
+  - type: IgnitionSource
+    temperature: 1200
+  - type: StaticPrice
+    price: 1000
+
+- type: entity
+  id: LPPPlasmaCartridge
+  parent: BaseItem
+  name: Plasma cartridge
+  description: "Special cartridges containing high-pressure plasma for injection into a specialized container of plasma sword. Disposable."
+  components:
+  - type: Item
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)